bcda/auth/middleware.go

```diff
@@ -58,48 +59,58 @@ func ParseToken(next http.Handler) http.Handler {
 
 		tokenString := authSubmatches[1]
 
-		token, err := GetProvider().VerifyToken(tokenString)
+		token, ad, err := AuthorizeAccess(tokenString)
 		if err != nil {
-			log.Auth.Errorf("Unable to verify token; %s", err)
 			handleTokenVerificationError(w, rw, err)
 			return
 		}
 
-		var ad AuthData
-		if claims, ok := token.Claims.(*CommonClaims); ok && token.Valid {
-			switch claims.Issuer {
```

Comment:
This switch case is unreachable because VerifyToken already does the issuer check here: 

https://github.com/CMSgov/bcda-app/blob/dd356ecda6aa85fb9befddc02b09a9e8f7e7907c/bcda/auth/ssas.go#L246

Reply:
LGTM! leaving a note:

VerifyToken() -> confirmTokenStringLegitimacy() -> confirmRequestorTokenPayload is path to issue check in linked comment